### ISSUE TITLE
Move documentation files to another folder

### ISSUE THREAD

**Issue by yong24s** [reposense/RepoSense#204]
Following `v1.11` requirements, the generated report folder will be changed from `reposense-report` to `docs`.

As our documentation files are also in `docs`, this impedes with development and testing.

Let's move our documentation files to another folder.

**Comment by damithc** [comment]
Not sure if this is necessary. The report should be in `target-folder/docs`, not necessarily `{root}/docs`

**Comment by damithc** [comment]
@yong24s is this still applicable?

**Comment by yong24s** [comment]
Yes, it is a good practice.

**Background:** 

We clone repositories into the root of our project for testing with assumption that git-clone **never** fails for positive tests. 

If the git-clone were to fail, the next git operation may get executed on our git working directory. This can potentially destroy our uncommitted changes.

**Comment by Tejas2805** [comment]
@yong24s Hi, have the instructions been included? Or can I work on this issue?

**Comment by yong24s** [comment]
@Tejas2805, yes you may work on this issue but I suggest you to take a look at #659 first, as at the issue is better defined.
